extract_lockscreen_images.py

```diff
@@ -2,6 +2,8 @@
 import shutil
 from pathlib import Path
 from datetime import datetime
+from PIL import Image
+from PIL.ExifTags import TAGS
```

Comment:
It doesn't look like `TAGS` is being used anywhere.

Reply:
Forgot to remove that MB.